operators/lazy-vs-eager-execution.md

```diff
@@ -0,0 +1,27 @@
+# Lazy vs Eager Executions
+
+Operators in Python have two modes of execution supported by Aqueduct.
+By default the mode of execution is `eager`. This can be configured via the `GlobalConfig` as shown below.
```

Comment:
But this is not the only way to configure it right? You can also run individual operators lazily with the `.lazy` annotate. Plus all the extract operators have a "lazy" argument.